WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -428,23 +418,46 @@ ZmContactSearchListView.prototype._getHeaderList =
 function() {
 	var headerList = [];
 	headerList.push(new DwtListHeaderItem({field:ZmItem.F_TYPE, width:ZmMsg.COLUMN_WIDTH_FOLDER_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name, width:ZmMsg.COLUMN_WIDTH_NAME_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_EMAIL, text:ZmMsg.email}));
-
+	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name}));
 	return headerList;
 };
 
+// Overriding the DwtListView.prototype.createHeaderHtml as we donot need a header, here.
+
+ZmContactSearchListView.prototype.createHeaderHtml =
+function() {
```

Comment:
By default, Js engine returns undefined values.
Going by suggestion, I have adjusted the changes.
https://github.com/Zimbra/zm-web-client/pull/117/commits/5b6db10fbe24b0d343ff59f3a79eb7fba550ea8b